Dockerfile

```diff
@@ -71,6 +71,10 @@ ENV	DATABASE_URL="postgresql://devto:devto@db:5432/PracticalDeveloper_developmen
 ENV DB_SETUP="false" \
   DB_MIGRATE="false"
 
+# In order for redis to work, these should be set
+ENV REDIS_URL="redis://redis:6379" \
```

Comment:
This assumes that people will always use Redis via Docker too. Some people may want to run the app via Docker, but certain services locally, I've certainly done that in the past. But those people can probably figure out how to get around this and we actually have the same assumption in `DATABASE_URL`.